app/resources/admin.html.template

```diff
@@ -68,12 +68,12 @@
     //find all languages that the user specified with "Use?" checkboxes and
     //their associated titles.  Only allows titles from used languages.
     var used_languages = [];
-    var languages_to_titles = {}
+    var languages_to_values = {};
+
     //there is a title row in the table, so the dropdowns are 1-indexed
-    for (var dropdown_index = 1; $('language_selection_' + dropdown_index) !=
+    for (var dropdown_index = 1; $('repo_titles_language_selection_' + dropdown_index) !=
```

Comment:
Please try to keep it under 80 characters.

Could you ask Ichikawa-san how to configure Emacs so that characters exceeding 80 chars are highlighted? I use vim.